lisav2: remove hardcoded root user for functional test cases

asofron opened the pull request:
**Hyper-V/ Ubuntu :**

[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/13/2018 20:35:16
[hyperv] VHD Under Test        : ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 4 (4 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:28
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 FCOPY-LARGE-FILE                                                   PASS                11.15 
[hyperv]     2 FCOPY-REPEATED-DELETE                                              PASS                 4.93 
[hyperv]     3 FCOPY-DISABLE-ENABLE                                               PASS                 9.38 
[hyperv]     4 FCOPY-NEGATIVE                                                     PASS                 3.38 

[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/13/2018 15:00:59
[hyperv] VHD Under Test        : ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 6 (6 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:45
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 KDUMP-CRASH-SINGLE-CORE                                            PASS                 8.82 
[hyperv]     2 KDUMP-CRASH-SMP                                                    PASS                 7.02 
[hyperv]     3 KDUMP-CRASH-NMI                                                    PASS                 7.87 
[hyperv]     4 KDUMP-CRASH-AUTO-SIZE                                              PASS                 6.66 
[hyperv]     5 KDUMP-CRASH-DIFFERENT-VCPU                                         PASS                 6.61 
[hyperv]     6 KDUMP-CRASH-16-CORES                                               PASS                 8.05 


[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/13/2018 15:23:03
[hyperv] VHD Under Test        : ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 5 (4 Pass, 0 Fail, 1 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:47
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 SQM-BASIC                                                          PASS                 2.72 
[hyperv]     2 STORAGE-VHDX-HOT-PLUG-UNPLUG-SCSI                                  PASS                 4.89 
[hyperv]     3 MAX-VCPU                                                           PASS                 6.72 
[hyperv]     4 TIMESYNC-VDSO                                                   Aborted                 2.52 
[hyperv]     5 STRESSTEST-CHANGE-MTU-RELOAD-NETVSC                                PASS                30.79 
[hyperv] TIMESYNC-VDSO :[INFO ] Info: Current VM Linux kernel version does not support vDSO feature.

[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/13/2018 15:47:11
[hyperv] VHD Under Test        : ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 2 (2 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:26
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 DYNAMIC-MEMORY-HOTADD-STRESS-APP                                   PASS                10.36 
[hyperv]     2 PAUSED-CRITICAL-HEARTBEAT                                          PASS                15.81 
[hyperv] 


[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/13/2018 16:34:21
[hyperv] VHD Under Test        : ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 2 (1 Pass, 0 Fail, 1 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:5
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 KVP-PUSH-KEY-VALUES                                                PASS                 2.71 
[hyperv]     2 KVP-MAX-READ                                                    Aborted                 2.43 
[hyperv] 
[hyperv] 2 KVP-MAX-READ :
[hyperv] 12/13/2018 16:39:12 : [INFO ] Kernels older than 3.10.0-862 require LIS-4.x drivers.
[hyperv] 12/13/2018 16:39:12 : [ERROR] Error: No LIS-4.x drivers detected. Skipping test.



[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/13/2018 16:05:52
[hyperv] VHD Under Test        : ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:14
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 BVT-CORE-RELOAD-MODULES-SMP                                        PASS                14.97 


[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 12/13/2018 16:24:18
[hyperv] VHD Under Test        : ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:1:45
[hyperv] XML File              : TestConfiguration
[hyperv] 
[hyperv]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[hyperv] ------------------------------------------------------------------------------------------------------
[hyperv]     1 STRESSTEST-RELOAD-MODULES-UP                                       PASS               105.42 


**Azure /Ubuntu :**


[azure] [LISAv2 Test Results Summary]
[azure] Test Run On           : 12/13/2018 16:18:20
[azure] ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
[azure] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[azure] Total Time (dd:hh:mm) : 0:0:3
[azure] XML File              : TestConfiguration
[azure] 
[azure]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[azure] ------------------------------------------------------------------------------------------------------
[azure]     1 BVT-VERIFY-KVP-DAEMON-STATUS                                       PASS                 3.22 
[azure] 

[azure] [LISAv2 Test Results Summary]
[azure] Test Run On           : 12/13/2018 16:27:28
[azure] ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
[azure] Total Test Cases      : 5 (5 Pass, 0 Fail, 0 Abort)
[azure] Total Time (dd:hh:mm) : 0:0:29
[azure] XML File              : TestConfiguration
[azure] 
[azure]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[azure] ------------------------------------------------------------------------------------------------------
[azure]     1 KDUMP-CRASH-SINGLE-CORE                                            PASS                 5.83 
[azure]     2 KDUMP-CRASH-SMP                                                    PASS                 6.09 
[azure]     3 KDUMP-CRASH-AUTO-SIZE                                              PASS                 5.88 
[azure]     4 KDUMP-CRASH-DIFFERENT-VCPU                                         PASS                 5.58 
[azure]     5 KDUMP-CRASH-16-CORES                                               PASS                 5.88 

[azure] [LISAv2 Test Results Summary]
[azure] Test Run On           : 12/13/2018 16:42:37
[azure] ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
[azure] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[azure] Total Time (dd:hh:mm) : 0:0:14
[azure] XML File              : TestConfiguration
[azure] 
[azure]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[azure] ------------------------------------------------------------------------------------------------------
[azure]     1 BVT-CORE-RELOAD-MODULES-SMP                                        PASS                14.45 
[azure] 
[azure] 


[azure] [LISAv2 Test Results Summary]
[azure] Test Run On           : 12/13/2018 17:08:31
[azure] ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
[azure] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[azure] Total Time (dd:hh:mm) : 0:1:45
[azure] XML File              : TestConfiguration
[azure] 
[azure]    ID TestCaseName                                                 TestResult TestDuration(in minutes) 
[azure] ------------------------------------------------------------------------------------------------------
[azure]     1 STRESSTEST-RELOAD-MODULES-UP                                       PASS               105.69 
[azure] 
[azure]

adriansuhov commented on Testscripts/Windows/STORAGE-HotUnplug.ps1:
```diff
@@ -74,7 +74,7 @@ function Main {
     $scsi=$true
     $REMOTE_SCRIPT="STORAGE-HotRemove.sh"
-    $rootUser="root"
+    $rootUser="$VMUserName"
```
Please remove $rootUser var and use $VMUsername directly in the commands. It's misleading now.

asofron replied:
fixed

adriansuhov commented on Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1:
```diff
@@ -31,7 +31,7 @@ function Main {
     )
 
-    $rootUser = "root"
+    $rootUser = "$VMUserName"
```
ditto

asofron replied:
Fixed

Commit:
Remove hardcoded root (Functional Tests)

Total test cases affected: 21
Tests that fail because of external causes:
- Timesync-VDSO - VDSO not supported by kernel
- KVP-MaxRead - Fails  get correct version if not RHEL distro

Libraries/CommonFunctions.psm1 - 2 TC (BVT-CORE-RELOAD-MODULES-SMP, STRESSTEST-RELOAD-MODULES-UP)
Testscripts/Windows/BVT-VERIFY-KVP-DAEMON-STATUS.ps1 -1 TC BVT-VERIFY-KVP-DAEMON-STATUS
Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1 - 1 TC DYNAMIC-MEMORY-HOTADD-STRESS-APP
Testscripts/Windows/FCOPY-DISABLE-ENABLE.ps1 - 1 TC FCOPY-DISABLE-ENABLE
Testscripts/Windows/FCOPY-LARGE-FILE.ps1 -1 TC FCOPY-DISABLE-ENABLE
Testscripts/Windows/FCOPY-NEGATIVE.ps1 -1 TC FCOPY-NEGATIVE
Testscripts/Windows/FCOPY-REPEATED-DELETE.ps1 - 1TC FCOPY-REPEATED-DELETE
Testscripts/Windows/Heartbeat-PausedCritical.ps1 - 1TC PAUSED-CRITICAL-HEARTBEAT
Testscripts/Windows/KDUMP-TestScript.ps1 6TC - KDUMP-CRASH-SINGLE-CORE , KDUMP-CRASH-SMP, KDUMP-CRASH-NMI,KDUMP-CRASH-AUTO-SIZE, KDUMP-CRASH-AUTO-SIZE ,KDUMP-CRASH-16-CORES
Testscripts/Windows/KVP-MaxRead.ps1 1 TC - KVP-MAX-READ
Testscripts/Windows/KVP-PushKeyValues.ps1 -1TC KVP-PUSH-KEY-VALUES
Testscripts/Windows/Max-vCPU.ps1 - 1 TC MAX-VCPU
Testscripts/Windows/SQM-Basic.ps1 - 1 TC SQM-BASIC
Testscripts/Windows/STORAGE-HotUnplug.ps1 - 1TC STORAGE-VHDX-HOT-PLUG-UNPLUG-SCSI
Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1 -1TC STRESSTEST-CHANGE-MTU-RELOAD-NETVSC
Testscripts/Windows/Timesync-VDSO.ps1 -1TC TIMESYNC-VDSO


fix
